src/index.tsx

```diff
@@ -1,3 +1,5 @@
 export * from './components/AnalyticsProvider';
+export * from './components/AnalyticsPageView';
 export * from './utils/googleAnalytics';
 export * from './contexts/useAnalyticsContext';
+export * from './hooks/useAnalyticsPageView';
```

Comment:
```suggestion
export * from './components';
export * from './contexts';
export * from './hooks';
export * from './utils';
```

디렉토리 별로 따로 barrel을 두고, `src/index.ts`에선 각 디렉토리의 index를 export만 하는 식으로 변경해보면 어떨까요.?!

Reply:
짱짱
applied in https://github.com/EveryAnalytics/react-analytics-provider/pull/41/commits/2204e9a0878caf5dae9aaa14467e33bf738c74f2